packages/nest/migrations.json

```diff
@@ -75,6 +75,31 @@
           "alwaysAddToPackageJson": true
         }
       }
+    },
+    "14.5.0": {
+      "version": "14.5.0-beta.1",
+      "packages": {
+        "@nestjs/common": {
+          "version": "^9.0.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@nestjs/core": {
+          "version": "^9.0.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@nestjs/platform-express": {
+          "version": "^9.0.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@nestjs/schematics": {
+          "version": "^9.0.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@nestjs/testing": {
+          "version": "^9.0.0",
+          "alwaysAddToPackageJson": false
+        }
+      }
```

Comment:
Do we need to include @nestjs/swagger here?

Reply:
Good catch. nestjs/swagger was indeed updated to v6. Thanks